src/ring_theory/power_series/basic.lean

```diff
@@ -1196,6 +1196,23 @@ rescale (-1 : A)
 @[simp] lemma eval_neg_hom_X : eval_neg_hom (X : power_series A) = -X :=
 rescale_neg_one_X
 
+/-- Two ways of truncating a power series are the same. -/
+lemma minus_const_eq_shift_mul_x (φ: power_series A) :
+  (φ - C A (constant_coeff A φ)) = power_series.mk (λ p, coeff A p.succ φ) * X:=
```

Comment:
```suggestion
lemma minus_const_eq_shift_mul_x (φ : power_series A) :
  φ - C A (constant_coeff A φ) = power_series.mk (λ p, coeff A p.succ φ) * X :=
```

Reply:
Please use `p + 1` instead of `p.succ`, because there are more simp lemmas about `p + 1`.

Reply:
ok, will do

Reply:
> Please use `p + 1` instead of `p.succ`, because there are more simp lemmas about `p + 1`.

That's useful to know!